### PR TITLE
as_numpy utility function

### PR DESCRIPTION
**Pull request by Olllom**
a basic utility function to avoid writing `.detach().cpu().numpy()` all the time

### CODE REVIEW

**Review by LeonKlein — APPROVED** [review]
Works like a charm
